moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryCouncilDistricts.js

```diff
@@ -28,22 +33,18 @@ const getAllCouncilDistricts = (projectGeography, childProjectGeography) => {
 /**
  * ProjectSummaryCouncilDistricts Component
  * @param {Object} data - The project geography from the graphql query's data object
- * @param {Object} classes - The shared style settings
  * @returns {JSX.Element}
  * @constructor
  */
 const ProjectSummaryCouncilDistricts = ({
   projectGeography,
-  classes,
   childProjectGeography,
 }) => {
   return (
-    <Grid item xs={12} className={classes.fieldGridItem}>
-      <Typography className={classes.fieldLabel}>
-        Council district(s)
-      </Typography>
-      <Box className={classes.fieldBox}>
-        <Typography className={classes.fieldLabelTextNoHover}>
+    <Grid item xs={12}>
```

Comment:
removed the `fieldGridItem` class here since the 'tags' and 'subprojects' sections below have their own top spacing and this was adding extra unnecessary spacing.